services/workflows-service/src/alert/alert.service.ts

```diff
@@ -140,8 +152,56 @@ export class AlertService {
     }
   }
 
-  // Specific alert check logic based on the definition
-  private async checkAlert(alertDefinition: AlertDefinition) {
+  async checkOngoingMonitoringAlert(businessReport: BusinessReport, businessCompanyName: string) {
+    const alertDefinitions = await this.alertDefinitionRepository.findMany(
```

Comment:
just use order by serveriy in here and you can remove the sort of alerts at line 199
<img width="781" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/a645e158-113a-4b97-b35b-34a218da6532">